deps/blas.mk

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS :=
+OPENBLAS_FFLAGS := $(FFLAGS) $(JFFLAGS)
 OPENBLAS_CFLAGS :=
```

Comment:
Yes, but thats just recovering the status pre #20806 . If anything, this patch adds `JFFLAGS` on all systems (which was not the case pre #20806 where on windows the set `FFLAGS` [here](https://github.com/JuliaLang/julia/blob/44eadf4149e12c6fab25552da59e425e52f7ff12/deps/blas.mk#L7) was overwritten [here](https://github.com/JuliaLang/julia/blob/44eadf4149e12c6fab25552da59e425e52f7ff12/deps/blas.mk#L58))